File: TrainingCshar/Load.vb
Namespace TrainingCshar
    Friend Class Load
        <STAThread>
        Private Shared Sub Main(ByVal args As String())
            System.Windows.Forms.Application.EnableVisualStyles()
            System.Windows.Forms.Application.SetCompatibleTextRenderingDefault(False)
            System.Windows.Forms.Application.Run(New Formulaio.FStart())
        End Sub
    End Class
End Namespace

File: TrainingCshar/Load.cs
using CsharLibrary;
namespace TrainingCshar
{
    internal class Load
    {
        [System.STAThread]
        private static void Main(string[] args)
        {
            var service = new Services();
            var container = service.csharContainer;

            System.Windows.Forms.Application.EnableVisualStyles();
            System.Windows.Forms.Application.SetCompatibleTextRenderingDefault(false);
            System.Windows.Forms.Application.Run(new Formulaio.FStart(container));
        }
    }
}